gallery/src/components/NxIconDropdown/NxIconDropdownExample.tsx

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+import { faEllipsisV } from '@fortawesome/free-solid-svg-icons';
+import { NxIconDropdown, useToggle, NxTextLink } from '@sonatype/react-shared-components';
+
+function NxIconDropdownNavigationExample() {
+  const [isOpen, onToggleCollapse] = useToggle(false),
+      deleteFn = () => { alert('delete'); },
+      onClick = () => { alert('click'); };
+
+  return (
+    <NxIconDropdown isOpen={isOpen}
+                    onToggleCollapse={onToggleCollapse}
+                    icon={faEllipsisV}
+                    title="Options">
+      <NxTextLink onClick={onClick} href="#/pages/NxDropdown" className="nx-dropdown-link">
+        Text link 1
+      </NxTextLink>
+      <NxTextLink onClick={onClick} href="#/pages/NxDropdown" className="nx-dropdown-link">
+        Text link 2 - this link should trigger truncation
+      </NxTextLink>
+      <NxTextLink external className="nx-dropdown-link" onClick={onClick}>Website Link</NxTextLink>
```

Comment:
The lack of `href` on this makes it not focusable

Reply:
> The lack of `href` on this makes it not focusable

Fixed here: https://github.com/sonatype/sonatype-react-shared-components/pull/465/commits/aa57355c2fb36d457c5a975613a5cb5f56b0c72a